packages/chat-component-bindings/src/hooks/usePropsFor.ts

```diff
@@ -56,6 +59,8 @@ export type GetSelector<Component extends (props: any) => JSX.Element | undefine
   typeof SendBox
 > extends true
   ? SendBoxSelector
+  : AreEqual<Component, typeof RichTextSendBox> extends true
```

Comment:
Should there be `/* @conditional-compile-remove(rich-text-editor) */`
for this line too?

Reply:
Ya I thought the same thing, took me awhie to figure out what was going on but that causes the entire block to be removed. I followed what was done for the Call-component-bindings version of usePropsFor.